x-pack/plugins/alerting/server/rules_client/methods/delete.ts

```diff
@@ -66,6 +67,13 @@ async function deleteWithOCC(context: RulesClientContext, { id }: { id: string }
     throw error;
   }
 
+  await untrackRuleAlerts(
```

Comment:
i tried deleting a rule with active alerts and the alerts stayed active.

Reply:
Wasn't aware that `delete` is deprecated and everything in the UI uses `bulk_delete` now. Latest commit implements untrack on `bulk_delete`